database/seeders/StudentTagsSeeder.php

```diff
@@ -0,0 +1,123 @@
+        "Cognitive Development" => ["cognitive development", "learning", "school readiness ", "skepticism", "skill", "skill learning"], 
+        "Cognitive Neuroscience" => ["brain training", "cognitive neuroscience", "EEG", "electrophysiology", "human neuroimaging"], 
+        "Developmental Psychology" => ["adults", "aging", "Alzheimer’s disease ", "child development", "children", "cognitive aging", "early intervention", "healthy aging", "infant", "lifespan", "motor development", "normal aging", "parent-child interaction", "parent-child relations", "parenting",  "poverty",  "sex differences",  "social development",  "typical"], 
+        "Health Psychology" => ["conflict", "coping", "drug discovery", "habit", "health", "immunology", "metabolism", "nutrition", "reconcile", "stress"], 
+        "Hearing / Audiology" => [ "audition", "auditory"], 
+        "Medical Devices & Assistive Technology" => [ "educational technology"], 
+        "Neuroscience / Neuropsychology" => ["affective neuroscience", "amyloid", "brain networks ", "Cultural Neuroscience", "fMRI", "hippocampus", "human DRG", "molecular", "mouse", "MRI", "neuroimaging", "neuropsychology", "neuroscience", "nociceptor", "pharmacology",  "plasticity",  "prefrontal cortex",  "social neuroscience", "synaptic plasticity",  "TMS",  "vagus nerve stimulation (VNS)"], 
+        "Pain" => [ "migraine", "pain"], 
+        "Personality Psychology" => [ "narcissism", "personality", "prosociality", "behavior", "personality pathology"], 
+        "Social Psychology" => ["body image", "body perception", "couples", "cross-cultural", "cultural diversity", "culture", "emotion", "emotions", "family", "interaction", "intimacy", "morality", "relationships", "signaling", "social communication",  "social trait perception",  "socioeconomic status"], 
+        "Speech / Language" => ["bilingual", "communication", "developmental language disorder", "language", "language development", "literacy", "motor", "semantic", "speech", "speech development", "speech perception", "speech therapy", "word learning"], 
+        "Speech and Language Disorders" => ["assessment", "disordered", "SLI"], 
+        "Statistics / Measurement / Modeling / Machine Learning" => ["artificial intelligence", "computational", "computational modeling", "dyadic data analysis", "eyetracking", "face recognition", "machine learning", "mathematical psychology", "psychometrics"], 
+    ];
+    
+    /**
+     * Run the StudentTags Seeder
+     *
+     * @return void
+     */
+    public function run()
+    {
+
+        $this->createNewTags();
+        $this->syncStudentApplicationsNewTags();
+
+    }
+
+    /**
+     * Create new tags with the type "App\StudentNew"
+     *
+     * @return void
+     */
+    public function createNewTags()
+    {
+        Tag::findOrCreate(collect($this::NEW_TAGS_ASSOCIATION)->keys(), "App\StudentNew");
```

Comment:
It's probably slightly more efficient to use the built-in PHP function `array_keys()` here instead of instantiating a Collection just to use its `keys()` method.